.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,35 @@
+name: CIFuzz
+on: [pull_request]
+permissions: {}
```

Comment:
Due to the way `pull_request` triggers work on GitHub, each time someone pushes changes to an open PR, GitHub will start another run of the workflow. If an ongoing run hasn't finished yet, this results in multiple simultaneously-running copies of the workflow (hardly ever useful). Consequently, we should add a `concurrency` property value to make sure that any ongoing runs are cancelled. 

```
permissions: {}

concurrency:
  # Cancel any previously-started but still active runs on the same branch.
  cancel-in-progress: true
  group: ${{github.workflow}}-${{github.event.pull_request.number||github.ref}}
```